ft_crossfrequencyanalysis.m

```diff
@@ -112,7 +112,9 @@
   cfg.chanlow   = ft_channelselection(cfg.chanlow, freqlow.label);
   cfg.chanhigh  = ft_channelselection(cfg.chanhigh, freqhigh.label);
   labelcmb = ft_channelcombination({cfg.chanlow,cfg.chanhigh},union(freqlow.label, freqhigh.label));
-  labelcmb(arrayfun(@(x) strcmp(labelcmb{x,:}), 1:size(labelcmb,1)),:) = []; % remove auto
+  % - remove duplicates
+  labelcmb = cellfun(@(cmb) strsplit(cmb,'|'),unique(arrayfun(@(x) strjoin(labelcmb(x,:),'|'), 1:size(labelcmb,1), 'UniformOutput', false)),'UniformOutput',false);
```

Comment:
Are there situations in which ft_channelcombination returns duplicates? I would expect that ft_channelcombination warrants that this is not the case. If - for some reason - ft_channelcombination is not robust enough to ensure this, in case it is called as in line 114, I think that the issue of duplicates should be solved within ft_channelcombination.